pysparkle/pysparkle/utils.py

```diff
@@ -33,3 +33,15 @@ def substitute_env_vars(input_str: str) -> str:
             input_str = input_str.replace("{" + placeholder + "}", env_var_value)
 
     return input_str
+
+
+def filter_csv_files(paths: list[str]) -> list[str]:
```

Comment:
Could make generic for any file type by taking `datasource_type` / `extension`argument.

Reply:
done